src/definitions/modules/slider.js

```diff
@@ -756,17 +756,19 @@ $.fn.slider = function(parameters) {
             value = ratio * (max - min) + min;
             return value;
           },
-          eventPos: function(event, originalEvent) {
+          eventPos: function(event) {
             if(module.is.touch()) {
               var
-                touchY = event.changedTouches[0].pageY || event.touches[0].pageY,
-                touchX = event.changedTouches[0].pageX || event.touches[0].pageX
+                touchEvent = event.changedTouches ? event : event.originalEvent,
+                touches = touchEvent.changedTouches[0] ? touchEvent.changedTouches : touchEvent.touches,
+                touchY = touches[0].pageY,
+                touchX = touches[0].pageX
               ;
               return module.is.vertical() ? touchY : touchX;
             }
             var
-              clickY = event.pageY || originalEvent.pageY,
-              clickX = event.pageX || originalEvent.pageX
+              clickY = event.pageY || event.originalEvent.pageY,
```

Comment:
Can we assume `originalEvent` always exists ?
If not, I am afraid that Null Pointer Exception.

Reply:
Yes, we can. pageX/pageY should always be available  in a non jquery event. Only the fallback is to use event.originalEvent which should only happen when jquery is used...and FUI still needs jquery anyway to make the slider work.
As of https://api.jquery.com/category/events/event-object/ we can rely on the event.originalEvent existance

Reply:
Understood 👍